File: src/DataStructure/Array/1D/Easy/PivotIndex/python/pivotIndex_test.py
import pytest
from PivotIndex.python.pivotIndex import Solution
from PivotIndex.python.testCases import runTestCase, cases


@pytest.mark.parametrize('case', cases)
def test(case):
    solution = Solution()
    assert runTestCase(solution.pivotIndex, case) == case['expected']

File: src/DataStructure/Array/1D/Easy/PivotIndex/javascript/pivotIndex.test.js
import runTestCase, { cases } from './testCases.js';

import findPivot from './pivotIndex.es3.cjs';

test.each(cases.map(c => [c.name, c]))(
  'Data Structure/Array/1D/Array/[724] Pivot Index - Case %#',
  (name, testCase) => expect(runTestCase(findPivot, testCase)).toEqual(testCase.expected),
);
